OpenRA.Mods.Common/Lint/CheckChromeHotkeys.cs

```diff
@@ -71,6 +87,20 @@ public void Run(Action<string> emitError, Action<string> emitWarning, ModData mo
 					}
 				}
 
+				// Check runtime-defined hotkey names
+				List<string> checkMethods;
+				var widgetType = node.Key.Split('@')[0];
+				if (customLintMethods.TryGetValue(widgetType, out checkMethods))
+				{
+					var type = modData.ObjectCreator.FindType(widgetType + "Widget");
+					var keyNames = checkMethods.SelectMany(m => (IEnumerable<string>)type.GetMethod(m).Invoke(null, new[] { node.Value }));
+
+					Hotkey unused;
+					foreach (var name in keyNames)
+						if (!namedKeys.Contains(name) && !Hotkey.TryParse(name, out unused))
```

Comment:
This works fine with HotkeyCount=0, but with missing HotkeyPrefix it passes lint but produces strange/wrong hotkeys. Ideally HotkeyPrefix should be mandatory for count>0.